protocol/communities/manager.go

```diff
@@ -4771,6 +4943,91 @@ func (m *Manager) createCommunityTokenPermission(request *requests.CreateCommuni
 	}
 
 	return community, changes, nil
+
+}
+
+func (m *Manager) UpdateControlNode(communityID types.HexBytes, controlNode types.HexBytes) (*Community, error) {
+	community, err := m.GetByID(communityID)
+	if err != nil {
+		return nil, err
+	}
+	if community == nil {
+		return nil, ErrOrgNotFound
+	}
+
+	pubKey, err := crypto.UnmarshalPubkey(controlNode)
+	if err != nil {
+		return nil, err
+	}
+
+	if community.ControlNode().Equal(pubKey) {
+		return community, nil
+	}
+
+	community.setControlNode(pubKey)
+	err = m.persistence.SaveCommunity(community)
+	if err != nil {
+		return nil, err
+	}
+
+	return community, nil
+}
+
+// FIXME: temporary for POC's tests, remove me once communities use timestamp hinted lamport clocks
```

Comment:
I think @osmaczko 's PR was merged, so maybe this can be removed?

Reply:
Yes, this should be removed.